pandas/tests/frame/test_repr_info.py

```diff
@@ -411,7 +411,7 @@ def test_to_records_with_na_record(self):
     def test_to_records_with_inf_as_na_record(self):
         # GH 48526
         expected = """   NaN  inf         record
-0  NaN    b    [0, inf, b]
+0  inf    b    [0, inf, b]
```

Comment:
If `use_inf_as_na` is enabled, shouldn't `inf` still be rendered as `NaN`?

Reply:
No, we still should render the "inf" accurately  Notice on L413 that we already do this when the inf is in the columns